cms/static/js/views/modals/course_outline_modals.js

```diff
@@ -317,43 +305,39 @@ define(['jquery', 'backbone', 'underscore', 'gettext', 'js/views/baseview',
                 'maxTime': '05:00',
                 'forceRoundTime': false
             });
+
+            this.setExamType(this.model.get('is_time_limited'), this.model.get('is_proctored_enabled'),
+                            this.model.get('is_practice_exam'));
             this.setExamTime(this.model.get('default_time_limit_minutes'));
-            this.setExamTmePreference(this.model.get('is_time_limited'));
+        },
+
+        setExamType: function(is_time_limited, is_proctored_enabled, is_practice_exam) {
+            
+            if (!is_time_limited) {
+                this.$("#id_not_timed").prop('checked', true);
+                return;
+            }
+
+            if (!is_proctored_enabled && !is_practice_exam ) {
```

Comment:
I think this should be in the if condition `if (this.options.enable_proctored_exams) {`